Tools/machines/lumi-csc/submit.sh

```diff
@@ -27,4 +25,19 @@ export FI_MR_CACHE_MONITOR=memhooks  # alternative cache monitor
 export ROCFFT_RTC_CACHE_PATH=/dev/null
 
 export OMP_NUM_THREADS=1
-srun ../warpx inputs > outputs
+
+cat << EOF > select_gpu
+#!/bin/bash
+
+export ROCR_VISIBLE_DEVICES=\$SLURM_LOCALID
+exec \$*
+EOF
+
+chmod +x ./select_gpu
```

Comment:
Potentially, we could add a `sleep 1` after FS operations like those, in case you see startup issues at scale.

Reply:
It is a good idea. I've added a couple of `sleep 1` , after `cat` and after `chmod +x`

Reply:
I think one after chmod is sufficient, because it's only the launch node that creates this and thus it will be aware of the file created to modify its rights.